services/lgtm/lgtm-alerts.service.js

```diff
@@ -1,71 +1,45 @@
 'use strict'
 
-const LegacyService = require('../legacy-service')
-const { makeBadgeData: getBadgeData } = require('../../lib/badge-data')
-const { checkErrorResponse } = require('../../lib/error-helper')
+const LgtmBaseService = require('./lgtm-base')
 const { metric } = require('../../lib/text-formatters')
 
-// This legacy service should be rewritten to use e.g. BaseJsonService.
-//
-// Tips for rewriting:
-// https://github.com/badges/shields/blob/master/doc/rewriting-services.md
-//
-// Do not base new services on this code.
-module.exports = class LgtmAlerts extends LegacyService {
-  static get category() {
-    return 'analysis'
+module.exports = class LgtmAlerts extends LgtmBaseService {
+  static get route() {
+    return {
+      base: 'lgtm/alerts/g',
+      pattern: ':user/:repo',
+    }
   }
 
-  static get route() {
+  async handle({ user, repo }) {
+    const { alerts } = await this.fetch({ user, repo })
+    let color = 'yellow'
+    if (alerts === 0) {
+      color = 'brightgreen'
+    }
```

Comment:
Assigning colours is a rendering concern. Can we reduce `handle()` to just

```js
const { alerts } = await this.fetch({ user, repo })
return this.constructor.render({ alerts })
```

and move the colours block to render()